src/Components/Blazor/Build/testassets/razorclasslibrary/RazorClassLibrary.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <ReferenceBlazorBuildLocally>true</ReferenceBlazorBuildLocally>
```

Comment:
I'm not sure this does anything in these projects. Here's how you import the targets instead:

https://github.com/aspnet/AspNetCore/blob/blazor-wasm/src/Components/Blazor/Build/testassets/standalone/standalone.csproj#L2

Reply:
You are right, this is left-over